.github/workflows/test.yml

```diff
@@ -1,8 +1,15 @@
-on: [push, pull_request]
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - '**'
```

Comment:
I wonder, is `'*'` not enough?

Reply:
This was intentional. `**` is, I think, more generally correct. Because a branch could be something like `a/b/c`, in which case `*` would not match:

https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet

Reply:
TIL, thanks